neat_periodic_task/lib/neat_periodic_task.dart

```diff
@@ -186,6 +190,8 @@ class NeatPeriodicTaskScheduler {
   /// This returns a [Future] that is completed when the scheduler has finished
   /// any on-going iterations. This will not abort an ongoing task, but it will
   /// stop further iterations of the task.
+  ///
+  /// Once stopped a [NeatPeriodicTaskScheduler] cannot be started again.
   Future<void> stop() async {
```

Comment:
Should we rename `stop()`->`close()` - seems more conventional, and would perhaps better indicate that you cannot restart.